cookbooks/ey-nginx/templates/default/fpm-server.conf.erb

```diff
@@ -85,7 +85,7 @@ server {
     fastcgi_intercept_errors    off;
     fastcgi_param               SCRIPT_FILENAME $document_root$fastcgi_script_name;
     fastcgi_param               SERVER_NAME $hostname;
-    <% if  File.exists?("/etc/newrelic/newrelic.cfg") %>
+    <% if  File.exists("/etc/newrelic/newrelic.cfg") %>
```

Comment:
Why is this one different than the others?

Reply:
It's my typo. Thanks for catching this. I fixed it.